### DISCUSSION TITLE
Define a fallback route when no Statamic routes apply

### DISCUSSION THREAD

**Discussion by tubsn** [statamic/cms#7608] [Q&A]
Hey guys I was wondering if anyone has an Idea how to create a "default" route (with a controller that includes a mappingtable for old SEO-URLs) which is only applied, when there is no "Statamic" route defined for that url.

eg:
www.url.de/collection/article 
(known Statamic Route - normal behavoir)
**-> default Statamic Controller**

www.url.de/my-old-link-with-seo-relevance 
(called only, when there is no article with the slug 'my-old-link-with-seo-relevance')
**-> MappingDatabase Controller**

I tried the laravel routing, but it gets called prior to the statamic routes.
Thanks for your help

**Comment by edalzell** [comment]
use the 404 template?

**Reply by tubsn** [reply]
hey thanks for the reply, I thought about that but I was hoping for a solution on the controller level :/

**Reply by edalzell** [reply]
Just had another thought you could make a middleware that checks to see if the content exists, then redirects appropriately when not?

It's a bit hacky cuz that code would be run twice (once in your middleware and once in the Statamic's FrontendController), but might work for you?

**Reply by tubsn** [reply]
I think I´ll try the errorpage solution maybe I can sneak myself in somewhere before the template gets rendered.
